tabnames.py

```diff
@@ -0,0 +1,35 @@
+import mysqlhack
+import org.bukkit as bukkit
+import json
+from java.util import UUID as UUID
+from helpers import *
+from org.bukkit import *
+from traceback import format_exc as trace
+from iptracker_secrets import *
+
+tabnames_version = "v1.0.0"

```

Comment:
The version tag is something we agreed on lately to enable the use of versioning. There already are modules that use the versioning system (like the new alias) to shut down themselfes in case of mismatching versions to prevent potential harm to the server